static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -651,16 +651,16 @@
 }
 
 .creative--ad-feature-mpu-large-v2 {
-    padding: 0 0 $gs-baseline / 2;
+    padding: 0;
 
     .creative__article {
         padding: 0;
 
         h2 {
             padding: $gs-baseline / 2;
-            font-size: get-font-size(bodyHeading, 3);
-            line-height: get-line-height(bodyHeading, 1);
-            height: 44px;
+            font-size: get-font-size(headline, 2);
+            line-height: get-line-height(headline, 1);
+            height: 36px;
```

Comment:
We have a variable `$gs-row-height` which is part of our grid system (_grid-system.scss) and is exactly 36px so we should use that